src/g42-main.cpp

```diff
@@ -426,10 +427,18 @@ int main(int argc, const char **argv) { // the main() function
 		_erro( mo_file_reader::gettext("L_unknown_exception_running_server") );
 		exit_code = 3;
 	}
-	if( !exception_catched )
-		_note(mo_file_reader::gettext("L_exit_no_error"));
+
+	try {
+		if( !exception_catched )
+			_note(mo_file_reader::gettext("L_exit_no_error"));
```

Comment:
please take expression in { ... } when after "if" there is a new line

if (...) {
  _note (...);
}